src/models/ollama.py

```diff
@@ -0,0 +1,66 @@
+import os
+import ollama
+from tqdm.contrib.concurrent import thread_map
+
+from src.models.llm import LLM
+from src.utils.mylogger import MyLogger
+
+_model_name_map = {
+    "ollama-qwen-coder": "qwen2.5-coder:latest",
+    "ollama-qwen": "qwen2.5:32b",
+    "ollama-llama3": "llama3.2:latest",
+    "ollama-deepseek-32b": "deepseek-r1:32b",
+    "ollama-deepseek-7b": "deepseek-r1:latest",
+}
```

Comment:
Right above `_OLLAMA_DEFAULT_OPTIONS` map, include a comment that links to the options parameter documentation 

https://github.com/ollama/ollama/blob/main/docs/modelfile.md#valid-parameters-and-values

Also mention the values of the keys are the default values and to alter as needed.